src/main/java/ulaval/glo2003/Seller.java

```diff
@@ -0,0 +1,127 @@
+package ulaval.glo2003;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.UUID;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Seller {
+    private final String name;
+    private final String birthdate;
+    private final String email;
+    private final String phoneNumber;
+    private final String bio;
+
+    private final String id;
+    private final String createdAt;
+
+    public Seller(String name, String birthdate, String email, String phoneNumber, String bio) {
+        this.name = name;
+        this.birthdate = birthdate;
+        this.email = email;
+        this.phoneNumber = phoneNumber;
+        this.bio = bio;
+
+        verificationOfSellerParameters();
+
+        id = UUID.randomUUID().toString();
+        createdAt = Instant.now().toString();
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public String getBirthdate() {
+        return birthdate;
+    }
+
+    public String getEmail() {
+        return email;
+    }
+
+    public String getPhoneNumber() {
+        return phoneNumber;
+    }
+
+    public String getBio() {
+        return bio;
+    }
+
+    public String getId() {
+        return id;
+    }
+
+    public String getCreatedAt() {
+        return createdAt;
+    }
+
+    private void verificationOfSellerParameters(){
```

Comment:
On pourrait renommer la fct "validateParameters" pour garder la forme "Verbe+ContenuMethode". Aussi, j'aurais tendance à déplacer les throw dans les is{x}Invalid (et les changer pour "validate{x}").

Reply:
ouais changer verification pour validate ferait du sens